apps/native-component-list/src/screens/EventsScreen.tsx

```diff
@@ -228,6 +228,9 @@ export default class EventsScreen extends React.Component<Props, State> {
   };
 
   render() {
+    if (!this.props.route.params?.calendar) {
```

Comment:
These duplicate with https://github.com/expo/expo/pull/35872, but we can land both of these